src/main/java/org/trigon/hardware/rev/spark/io/RealSparkIO.java

```diff
@@ -66,7 +66,9 @@ public void configure(SparkBaseConfig configuration, SparkBase.ResetMode resetMo
 
     @Override
     public void setInverted(boolean inverted) {
-        motor.setInverted(inverted);
+        final SparkMaxConfig configuration = new SparkMaxConfig();
+        configuration.inverted(inverted);
+        motor.configure(configuration, SparkBase.ResetMode.kNoResetSafeParameters, SparkBase.PersistMode.kNoPersistParameters);
```

Comment:
Woudln't this reset the current config?
Is there really no easier way?

Reply:
no. There isn't a better way. As said [here](https://github.com/wpilibsuite/2025Beta/discussions/27#discussioncomment-11217925)